pkg/shell/vm.go

```diff
@@ -29,13 +28,16 @@ import (
 // types of errors, Run prints them using a custom format. For other
 // types of errors, their respective Error method is used for printing.
 func Run(command string) {
-	cmd, args := parser.Parse(command)
-	err := dispatch(cmd, args)
+	cmd, err := parser.Parse(command)
+	if err != nil {
+		fmt.Println(err)
+	}
+	err = dispatch(cmd)
```

Comment:
Maybe we could have a check here whether this command exists or something.

Reply:
Isn't it done below in the error checking?

Reply:
Isn't it done below in the error checking?